### DISCUSSION TITLE
Multiple OpenDTU instances ?

### DISCUSSION THREAD

**Discussion by mathieucarbou** [tbnobody/OpenDTU#534] [Q&A] [answered]
I am really having troubles reaching my inverters so I am trying different boards / combinations.

Is is possible to upload the SAME configuration on 2 or more OpenDTU instances, all communicating with the same inverters, but only one enabled for MQTT and Home Assistant ?

Thanks!

**Comment by smarthausleben** [comment, marked as answer]
1.	Yes you can access your inverter with 2 or more OpenDTU instances **but** you will face some disturbing readings if they try it at the same time.
2.	Having MQTT active on all devices should not be a Problem at all. Parallel requests to the inverter is the real problem because your will get wrong values and errors if the requests are coming at the same time.

If you have trouble to get readings, you may have a look here [How to troubleshoot OpenDTU instances not receiving data ?](https://github.com/tbnobody/OpenDTU/discussions/535#discussion-4787139) . Maybe some capacitors may solve your problem too.

**Reply by smarthausleben** [reply]
Just figured out you are also the author of the other question I was linking to 😂

**Reply by mathieucarbou** [reply]
I experienced the weirds readings yes... Voltage values and energy were completely unreasonably high ;-)
